src/webhooks/handleSubscriptionCanceled.ts

```diff
@@ -32,15 +33,19 @@ export default async function handleSubscriptionCanceled(
   if (productType === UserType.Business) {
     return usersService.destroyWorkspace(uuid);
   }
- 
+
+  if (hasBoughtALifetime) {
+    // This user has purchased a lifetime plan by having a subscription.
```

Comment:
_This user has switched from a subscription to a lifetime, therefore we do not want to downgrade his space_